### DISCUSSION TITLE
10월 3주차

### DISCUSSION THREAD

**Discussion by healtheloper** [Co-Studo/Co-Studo-front#37] [Weekly meeting]
### Polls 마감 시점 - Park
- 과반수(3명) 이상 + 1일 경과 시
- 2명 이상 + 반대의견X + 2일 경과시

### Firebase auth 도입 - Park
- Firebase 에서 제공하는 signIn ~~ 은 client 에서 사용하기를 권장 ([stackoverflow](https://stackoverflow.com/questions/46626585/how-to-authenticate-a-firebase-user-from-server-without-client-side-auth))
  - login 시 브라우저 내부에 토큰이 저장되는 방식이라던지, 현재 로그인 된 currentUser 를 받아오는 방식등의 메서드가 client 와 연동되어있음
- Firebase 를 통한 BFF 방식을 유지하되, auth 관련해서만 부분적 도입이 필요해보임

### Component 사용 시 의견 전달 방식 - Park
- 각자 만든 Component 를 사용하다가 생기는 의문 + 개선했으면 하는 점을 위한 Discussuon tab 생성?

### CSS Prop 방식 과 component.styled.tsx 분리 - Park, Jamie, Dony

**Comment by jindonyy** [comment]
[회의 정리 notion](https://costudo.notion.site/10-2-Polls-PR-Firebase-auth-CSSProps-vs-Styled-4c51b31de9a54d8db2b6427305d001b3)
